src/components/ListOfUsers.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./App.css";
+import users from "../users";
```

Comment:
Better import the users in the index.js and then pass them down as props. That way this component would be reusable... Right now you have a hard dependency in there.